botocore/config.py

```diff
@@ -256,6 +262,7 @@ class Config:
             ('tcp_keepalive', None),
             ('request_min_compression_size_bytes', None),
             ('disable_request_compression', None),
+            ('sigv4a_signing_region_set', None),
             ('client_context_params', None),
```

Comment:
Was there a reason we're changing order for existing entries? If not, we should append to the end.
```suggestion
            ('client_context_params', None),
            ('sigv4a_signing_region_set', None),
```